inc/namespace.php

```diff
@@ -432,7 +432,7 @@ function add_anchor_attribute( string $html ): string {
 	}
 
 	// Save the HTML changes.
-	$content = utf8_decode( $dom->saveHTML( $dom->documentElement ) ); //phpcs:ignore
+	$content = mb_convert_encoding( $dom->saveHTML( $dom->documentElement ), 'ISO-8859-1', 'UTF-8' ); //phpcs:ignore
```

Comment:
https://github.com/mtoensing/simpletoc/commit/fbe1c5262494f0a297c1f85abfac750bcca8e8c8 makes more sense to me.